modules/doc/content/getting_started/installation/offline_installation.md

```diff
@@ -119,6 +110,7 @@ tar -pzcf offline.tar.gz offline
 
 !alert note
 If operating on a Macintosh machine, creating a tarball to be extracted on a Linux machine produces warnings. They are warnings only, and can be safely ignored.
+!alert-end!
```

Comment:
I see. Thanks

Reply:
This is still present it would seem.
[View site here](https://mooseframework.inl.gov/docs/PRs/15765/site/getting_started/installation/offline_installation.html)